extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -104,6 +103,11 @@ class AmpStickyAd extends AMP.BaseElement {
       this.deferMutate(() => {
         toggle(this.element, true);
         this.viewport_.addToFixedLayer(this.element);
+        // Add border-bottom to the body to compensate space that was taken
+        // by sticky ad, so no content would be blocked by sticky ad unit.
+        const borderBottom = this.element./*OK*/offsetHeight;
```

Comment:
It's fine to start with just `Viewport`. But this is the question for you: it depends on what testing will show. If we have any problems with one type of the viewport vs the other - we'll have to push it deeper into ViewportBinding.